src/components/wallet/AddWallet.vue

```diff
@@ -181,6 +181,7 @@
                 ],
                 mnemonic:[
                     v => !!v || 'Mnemonic is required',
+                    v => (v && v.split(" ").length >= 15) || 'Mnemonic must be greater than 15 words',
```

Comment:
Lets rephrase to `Mnemonic must be at least 15 words` 
Since its `>=` we don't want to say greater. 

Another thought is we could be specific. I'm fairly confident that the only valid lengths are 15, 18, 21, and 24. It has to be between 15 and 24 and I believe divisible by 3.

@rcmorano Is my mnemonic requirement correct?

Reply:
TBH, I'm not sure about the specifics requirements about them, but we also have some 12-words mnemonics out there we might want to support (tho again, I'm not sure we are even able to do it with current `cardano-address` implementation).

I'd stick to daedalus' requirements just to play safe in these (unknown for me) waters heh

![image](https://user-images.githubusercontent.com/55876/106746338-ad755080-6622-11eb-9fe6-5360d3cea780.png)

![image](https://user-images.githubusercontent.com/55876/106746299-a3ebe880-6622-11eb-9786-6a50342ed72f.png)

Reply:
It looks like we could support all these:

```
$ echo vote shed jungle taste exhibit problem that hundred stamp cash ready pull river | cardano-address key from-recovery-phrase Shelley
user error (Invalid number of words: 9, 12, 15, 18, 21 or 24 words are expected.)
```

Reply:
the thing is with 9 words it doesn't work at all.

Reply:
I think 15, 18, 21, and 24. will be good